Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 # StackStorm version which refers to Docker images tag
 appVersion: "3.8"
 name: stackstorm-ha
-version: 0.110.0
+version: 0.111.0-rc0
```

Comment:
Would you please remove this? We'll bump the version at release time.

Reply:
Done. That is used for my own testing, and why this pr is still in draft